### DISCUSSION TITLE
Custom statusbar in 0.3 version

### DISCUSSION THREAD

**Discussion by og900aero** [sxyazi/yazi#1437] [1. Q&A]
I used a code (in init.lua) to rearrange and recolor the status bar in version 0.2.5, but it absolutely does not work anymore in version 0.3. What do I need to do to get it working again?I used the following code:

```
function Status:render(area)
  self.area = area

  local left = ui.Line { self:permissions(), self:owner(), self:modified() }
  local right = ui.Line { self:position() }
  return {
  ui.Paragraph(area, { left }),
  ui.Paragraph(area, { right }):align(ui.Paragraph.RIGHT),
  table.unpack(Progress:render(area, right:width())),
  }
end
```

**Comment by xfzv** [comment]
See https://github.com/sxyazi/yazi/discussions/1288#discussioncomment-10033306

**Reply by og900aero** [reply]
> See [#1288 (comment)](https://github.com/sxyazi/yazi/discussions/1288#discussioncomment-10033306)

Thx! I modified the code based on this and it works perfectly now.
